adafruit_ntp.py

```diff
@@ -94,6 +94,16 @@ def datetime(self) -> time.struct_time:
             poll = struct.unpack_from("!B", self._packet, offset=2)[0]
             self.next_sync = destination + (2**poll) * 1_000_000_000
             seconds = struct.unpack_from("!I", self._packet, offset=PACKET_SIZE - 8)[0]
+
+            # value should always be larger; giving a small buffer to handle jitter.
+            if (seconds + 5) < self._monotonic_start:
+                failed_offset = (self._monotonic_start - seconds) / 1_000_000_000
```

Comment:
Why `/ 1_000_000_000`? isn't monotonic_start and seconds in seconds units already? I think it would be better to just detect zero and either ignore it or raise an exception. poll should probably be ignored if seconds is zero too.

Reply:
I think I misread the code below it for `/ 1_000_000_000`.

Not ignoring `poll` because of where this test appears in the code, and I'm not sure what the poll value should be if it's ignored- the test in #37 would have given it a reasonable minimum value.